pms/tests/test_pms_folio_invoice.py

```diff
@@ -162,7 +157,6 @@ def test_invoice_partial_folio_by_steps(self):
 
     def test_invoice_partial_folio_diferent_partners(self):
         # ARRANGE
```

Comment:
a short descriptive text of the task of the test can help a lot to understand what is being tested